### DISCUSSION TITLE
Comparison of same trips

### DISCUSSION THREAD

**Discussion by riesi77** [teslamate-org/teslamate#1034] [Ideas]
**What would you like to be added**:
I wish I would have a dashboard where I could compare trips to the same destination.

**Why is this needed**:
I go to work every day with my car. 20km to go there and 20km back. In the meantime I have around 100 of these trips recorded.
I would like to compare these trips - especially on several criterias:

- Date based (maybe my driving improved over the time and I got lower consumtion rates)
- Temperature based (how much raises the consumption when winter comes)
- Average Speed (The faster I go, the higher consumption)
- Usage of heating, AC (How does it compare if I use the heating or AC compared to if I don't use it)

The output? I am not sure. Definitely the consumption. Maybe also the average speed could be interesting (do I drive faster in summer?). 

So we would need some kind of selection - maybe a list of trips that have the same start- and endpoint and have been documented for more than 5 times.
And then the definition of a graph. y axes the consumption, x axes takes the trips chronologically, and curves for the criterias like temperature, avg. speed,...

I do not know at all if this is feasable with the current Grafana Engine. And I think the concept also needs some shaping. But I hope you get the point (to get statistics based on the same driven way)

**Additional context**:

**Comment by riesi77** [comment]
Any feedback?

**Comment by DrMichael** [comment]
Well, you could go into Statistics and sort by the respective column...

**Reply by riesi77** [reply]
I'm not sure I understand. How can I filter for same trips? For instance Home-Work? On statistics, I only have a time based filter.

**Comment by DrMichael** [comment]
Ah, right. Perhaps a filter for start and destination on the Drives dashboard would help...

**Comment by DrMichael** [comment]
Have a look at this version of Drives. It allows you to filter Start and Destination with Geofences.
[Drives.zip](https://github.com/adriankumpf/teslamate/files/6252323/Drives.zip)
